data/net/minecraft/world/item/alchemy/PotionUtils.mapping

```diff
@@ -53,10 +53,10 @@ CLASS net/minecraft/world/item/alchemy/PotionUtils
 		ARG 0 compoundTag
 			COMMENT the passed {@code CompoundTag}
 	METHOD getPotion (Lnet/minecraft/world/item/ItemStack;)Lnet/minecraft/world/item/alchemy/Potion;
-		COMMENT Attempts to get the {@code Potion} from an {@code ItemStack}
-		COMMENT If it fails, returns the default one : {@code Potions.WATER}
+		COMMENT Attempts to get the {@link net.minecraft.world.item.alchemy.Potion Potion} from an {@link net.minecraft.world.item.ItemStack ItemStack}
+		COMMENT If it fails, returns {@code net.minecraft.world.item.alchemy.Potions#EMPTY Potions#EMPTY}
```

Comment:
As a matter of consistency, could you please apply the javadoc (modified accordingly) to the `getPotion(CompoundTag)` method?